airflow/config_templates/default_airflow.cfg

```diff
@@ -256,6 +256,15 @@ dag_processor_manager_log_location = {AIRFLOW_HOME}/logs/dag_processor_manager/d
 # Default to use task handler.
 task_log_reader = task
 
+[secrets_backend]
+# Name of secrets backend to enable (will precede env vars and metastore in search path)
+class_name =
+
+# The config_json param is loaded into a dictionary and passed to secrets backend __init__.
+# See documentation for the secrets backend you are using. JSON is expected.
+# Example for AWS SSM: {{"prefix": "/airflow", "profile_name": "my_aws_profile"}}
+config_json =
```

Comment:
And this to `client_kwargs` ??

Reply:
other then that I think it is ready to get merged 🎉

Reply:
👍